pkgs/by-name/co/compiledb/package.nix

```diff
@@ -7,18 +7,21 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "compiledb";
-  version = "0.10.1";
+  version = "0.10.7";
   pyproject = true;
 
-  build-system = [ python3Packages.setuptools ];
-
   src = fetchFromGitHub {
     owner = "nickdiego";
     repo = "compiledb";
-    tag = "v${version}";
-    hash = "sha256-pN2F0bFT5r8w+8kZOP/tU7Cx1SDN81fzkMfnj19jMWM=";
+    tag = version;
+    hash = "sha256-toqBf5q1EfZVhZN5DAtxkyFF7UlyNbqxWAIWFMwacxw=";
   };
 
+  build-system = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
   dependencies = with python3Packages; [
```

Comment:
Please drop `shutilwhich` from `dependencies`: https://github.com/nickdiego/compiledb/compare/v0.10.1...0.10.7#diff-bbb7c32237065bc7fba8d8b2118bb0064854a3c82c20e067519e8cc813f61cf4

Reply:
Nice spot 😄 it wasn't viewable in the diff